action.yml

```diff
@@ -129,6 +132,16 @@ runs:
         tutor local exec cms pip install -e /openedx/${{ inputs.app_name }}/
       shell: bash
 
+    - name: Install Inline Tutor plugin
```

Comment:
> I'm thinking we should execute this step right after installing the indexed plugins

I'll try it, but it didn't work. For example, the Middleware list is updated [here](https://github.com/eduNEXT/eox-core/pull/298/files#diff-f76c2bd29fed557218fb6db1da2eec5209876bbfb1bc90a8e3aacbd6ecce36d0R1-R7), but it needs eox-core to be installed first to work. That's why I added it after the plugin installation step.

> Also, if I remember correctly, in some cases it’s necessary to run tutor config save after enabling a plugin. Do you know if this is true?

Since [Tutor v17](https://github.com/overhangio/tutor/releases/tag/v17.0.0) this is not neccessary.

Reply:
I think it's good as it is then, thank you!